_posts/patterns/2015-04-15-icons.md

```diff
@@ -899,7 +921,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Visible, Show
+            eye
```

Comment:
That's a good point. "eye" isn't really a very good semantic name. How about I call this one "show" and the other one "hide"? I could leave "eye" as a synonym too.